Automattic/woocommerce-payments
# Exclude estimated deposits from useDeposits data hook by default

Fixes #7604

#### Changes proposed in this Pull Request

This PR filters out by `estimated` deposits from the **Payments → Deposits** screen by excluding the `estimated` deposits from the `useDeposits` data hook by default.

The `useDeposits` hook is only used by the DepositsList component (focus of this PR) and the [`useRecentDeposits`](https://github.com/Automattic/woocommerce-payments/blob/3da522158557512044209af0b48d8321d3c4a3a5/client/components/deposits-overview/hooks.ts#L20) hook, which explicitly filters out `estimated` deposits anyway.

#### Testing instructions





* Ensure your store has an estimated deposit by purchasing a product and setting the deposit schedule to daily, weekly or monthly.
* Visit the **Payments → Deposits** screen.
* Ensure no `Estimated` deposits appear in the list.
* Ensure the summary at the bottom of the table is correct.

-------------------


**Post merge**
